pkg/ddl/backfilling_dist_executor.go

```diff
@@ -156,6 +151,20 @@ func (s *backfillDistExecutor) Init(ctx context.Context) error {
 	return nil
 }
 
+func decodeIndexUniqueness(job *model.Job) (bool, error) {
+	unique := make([]bool, 1)
+	err := job.DecodeArgs(&unique[0])
+	if err != nil {
+		err = job.DecodeArgs(&unique)
+	}
+	if err != nil {
+		return false, errors.Trace(err)
+	}
+	// We only support adding multiple unique indexes or multiple non-unique indexes,
```

Comment:
Seems this limitation is too strong, normally, one table only will have one or two uk.

Reply:
It does not limit how many indexes a table can have. This means if there is a SQL like `add index i1, add unique index i2`, it will fallback to add them sequentially.